content/post/2017-05-22-ewenme-site.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2017-05-22
+author: Ewen Henderson
+title: Welcome to ewenme (you-n-me)
+tags: 
+ - Hugo
+ - Blogdown
+ - Netlify
+description: An intro to Ewen's new website
+---
+
+## A little bit on me
+
+I'm Ewen, an analyst and data science hopeful with a social sciences background, currently working in UK government. I'm about to join the [Bank of England](http://www.bankofengland.co.uk/Pages/home.aspx), the central bank for the United Kingdom, having been working for [Transport for London](https://tfl.gov.uk/) (government body responsible for the transport system in London) for a while.
+
+## Why start a site?
+
+I've been using R for data science-type workflows pretty intensively for about a year-and-a-half, and I had the realisation a few months back that I felt ready to add something to this conversation. I was also listening a lot to a journalism podcast, [Longform](https://longform.org/podcast), and they made writing sound quite noble/fun. Unfortunately for me, this realisation came before I discovered blogdown, so at first I put together a site using [Github Pages](https://pages.github.com/) and [Jekyll](https://jekyllrb.com/) (boo - the source files will remain over at this [Github repo](https://github.com/ewenme/ewenme.github.io)). 
+
+After coming across Yihui's tour-de-force `Blogdown` package soon after (I think it was in [this rstudio::conf talk](https://www.rstudio.com/resources/videos/advanced-r-markdown-tutorial/)), I knew that jumping across to this set-up sooner rather than later would save me time and make my new blogging life a good deal nicer. So, I wasted no time...
+
+## Going live
+
+My website [ewenme](http://ewenme.rbind.io/) (pronounced you-n-me) is now live, and part of the Rbind family (source files over [here](https://github.com/rbind/ewenme). It's hosted over at [Netlify](https://www.netlify.com/), which (I'm sure everyone has said already) was easy to get going with (and integrate with Git). 
+
+## The theme
+
+It was built using the [hugo-creative-portfolio-theme](https://github.com/kishaningithub/hugo-creative-portfolio-theme) by [Kishan B](https://github.com/kishaningithub), powered by Blogdown, and has a few things over there just now. The theme itself is probably intended for use by designer types, but I quite liked the idea of subverting this portfolio style for the purposes of an analytics blog. This is also a good moment to thank Amber (thanks, Amber!) for [this sick guide](https://proquestionasker.github.io/blog/Making_Site/) that helped keep me mostly calm throughout this process.
+
+## What now?
+
+I intend to keep the 'brief' pretty simple (for now) - that is, write and think about (and advocate for) data science in an opinionated but humane, ethical way, and to document the diversions I take in this field. I hope you can pick up something worthwhile on the way (there should be memes and puns worth stealing, at the very least). 
```

Comment:
I would quote the word "stealing" or replace it with something like "borrow". Just a suggestion.